SQL/0000-00-00-schema.sql

```diff
@@ -1643,6 +1671,386 @@ CREATE TABLE `parameter_type_override` (
   PRIMARY KEY (`Name`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 
+-- ********************************
+-- biobank tables
+-- ********************************
+                                                                                
+/*Global*/                                                                      
+CREATE TABLE `biobank_reference_table` (                                        
+  `ReferenceTableID` integer unsigned NOT NULL AUTO_INCREMENT,                  
+  `TableName` varchar(50) NOT NULL,                                             
+  `ColumnName` varchar(50) NOT NULL,                                            
+  CONSTRAINT `PK_biobank_reference_table` PRIMARY KEY (`ReferenceTableID`),     
+  CONSTRAINT `UK_biobank_reference_table_TableName_ColumnName` UNIQUE(`TableName`, `ColumnName`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+CREATE TABLE `biobank_datatype` (                                               
+  `DatatypeID` integer unsigned NOT NULL AUTO_INCREMENT,                        
+  `Datatype` varchar(20) NOT NULL UNIQUE,                                       
+  CONSTRAINT `PK_biobank_datatype` PRIMARY KEY (`DatatypeID`)                   
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+CREATE TABLE `biobank_unit` (                                                   
+  `UnitID` integer unsigned NOT NULL AUTO_INCREMENT,                            
+  `Label` varchar(20) NOT NULL UNIQUE,                                          
+  CONSTRAINT `PK_biobank_unit` PRIMARY KEY (`UnitID`)                           
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;                                        
+                                                                                
+                                                                                
+/*Container*/                                                                   
+                                                                                
+CREATE TABLE `biobank_container_capacity` (                                     
```

Comment:
I think this should be in the `biobank_container_type` directly. Does container types need to share common dimensions?

Reply:
Yes, there may several test tubes (redtop, greentop, etc.) that are used for different purposes but are each 10ml.

Reply:
But if you want to change the green ones for 11ml then you will have to create a new biobank_container_capacity or change the existing one (changing the red capacity in the same time). I still don't see why capacity should be extracted from container_type

Reply:
If you change a green one from 10ml to 11ml you would have to create a new container type that references a new value 11ml in the capacity table. You would not change the value 10 in the capacity table. 

I believe this decision was made to reduce redundancy as well. 

@ridz1208

Reply:
I think that was the reason, that some capacities like 10ml, 15ml will be used in a bunch of places but looking at it now, it might be an overkill.